Seasons Defense/Assets/Scripts/Enemy Missile.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyMissile : MonoBehaviour
+{
+    public GameObject explosionPrefab;
+    
+    GameObject[] civilizationPrefab;
+    private Transform target;
+    public float speed = 10f;
+    // Start is called before the first frame update
+    void Start()
+    {
+        civilizationPrefab = GameObject.FindGameObjectsWithTag("Civilization");
+        target = civilizationPrefab[Random.Range(0, civilizationPrefab.Length)].transform;
```

Comment:
Let's move this logic out of each enemy's missile. The creation of an external object (GameManager) and script's (SpawnManager.cs) will help to reduce the number of times this search is performed to a single time at game Start. Have the EnemySpawner and Enemy scripts call a public function for getting a target location for their missiles.